samples/Standalone.MvcSample/Areas/BankIdAuthentication/Views/BankId/_Login.cshtml

```diff
@@ -0,0 +1,12 @@
+﻿@using Microsoft.Extensions.Localization
+@model ActiveLogin.Authentication.BankId.AspNetCore.Areas.BankIdAuthentication.Models.BankIdLoginViewModel
+@inject IStringLocalizer<ActiveLogin.Authentication.BankId.AspNetCore.BankIdAuthenticationHandler> Localizer
+@addTagHelper *, Microsoft.AspNetCore.Mvc.TagHelpers
+
+<div class="bankid-login" style="width: 100%; max-width: 320px; margin: 0 auto; text-align: center; position: absolute; top: 50%; left: 50%; transform: translate(-50%, -50%);">
+    <div class="h3 mb-3 font-weight-normal">_Login partial from MvcSample</div>
```

Comment:
I'd like the sample still to use `Localizer` to show this usecase, maybe we can instead add a subheading or something similar?